packages/scripts/src/views/ListScripts.vue

```diff
@@ -0,0 +1,141 @@
+          <font-awesome-icon icon="edit" size="lg" />
+        </b-button>
+      </template>
+      <template
+        slot="remove"
+        slot-scope="data">
+        <b-button class="removeButton" size="sm" variant="danger" @click="doOpenModal(data.item.name)">
+          <font-awesome-icon icon="trash" size="lg" />
+        </b-button>
+      </template>
+      <template
+        slot="name"
+        slot-scope="data">
+        {{ data.item.name }} <b-badge>{{ data.item.type.name }} </b-badge>
+      </template>
+      <template
+        slot="parameters"
+        slot-scope="data">
+        <span v-for="(parameter, key) in data.item.parameters" :key="`parameter-${key}`"><span v-if="key!=0">,</span> {{ parameter.name }}</span>
+      </template>
+      <template
+        slot="execute"
+        slot-scope="data">
+        <ExecuteButton v-if="loaded" size="sm" :parameters="simpleParameters(data.item.parameters)" :name="data.item.name">Run</ExecuteButton>
+      </template>
+    </b-table>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue'
+import { mapState } from 'vuex'
+import { ReadyState } from '@/types/state'
+import ExecuteButton from '../components/ExecuteButton'
+
+export default Vue.extend({
+  name: 'ScriptList',
+  data () {
+    return {
+      fields: {
+        edit: {
+          label: '',
+          'class': 'compact align-middle'
+        },
+        remove: {
+          label: '',
+          'class': 'compact align-middle'
+        },
+        name: {
+          label: 'Name',
+          sortable: true,
+          'class': 'align-middle'
+        },
+        parameters: {
+          label: 'Parameters',
+          sortable: false,
+          'class': 'align-middle'
+        },
+        resultFileExtension: {
+          label: 'Result file extension',
+          sortable: false,
+          'class': 'compact align-middle'
+        },
+        execute: {
+          label: '',
+          sortable: false,
+          'class': 'compact'
+        }
+      },
+      hover: true,
+      showRemoveModal: false,
+      confirmedToRemove: '',
+      filter: null
+    }
+  },
+  computed: {
+    ...mapState(['scripts', 'loaded'])
+  },
+  components: { ExecuteButton },
+  methods: {
+    editScript (record) {
+      this.$router.push({ name: 'editscript', params: { id: record.name } })
+    },
+    simpleParameters (parameters) {
+      return parameters.map((parameter) => {
+        return parameter.name
+      })
+    },
+    doOpenModal (toRemove) {
+      this.confirmedToRemove = toRemove
+      this.showRemoveModal = !this.showRemoveModal
+    },
+    confirmedRemove () {
+      this.$store.dispatch('removeScript', this.confirmedToRemove)
+      this.confirmedToRemove = ''
+    },
+    createNewScript () {
+      this.$router.push({ name: 'newscript' })
+    }
+  }
+})
+</script>
+
+<style scoped>
+  >>> .compact {
```

Comment:
why are these details needed , is this going to work work with the theming ?

Reply:
`.compact{}` is used to make sure that the row is as small as it can be.
`tr {}` I do enable bootstraps 'hover' table style, but i also want the cursor to change. Not just the hovered row to lit up.
`.badge` the smaller badge i use for the variables don't sit nice, this will make it look better.

All changes are scoped and i don't see any theming problems.